src/main/java/teammates/ui/controller/CreateImageUploadUrlAction.java

```diff
@@ -0,0 +1,40 @@
+package teammates.ui.controller;
+
+import com.google.appengine.api.blobstore.BlobstoreFailureException;
+
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.util.Const;
+import teammates.common.util.GoogleCloudStorageHelper;
+
+public class CreateImageUploadUrlAction extends Action {
+
+    @Override
+    protected ActionResult execute() throws EntityDoesNotExistException {
+        return createAjaxResult(getCreateImageUploadUrlPageData());
+    }
+
+    protected final CreateImageUploadUrlAjaxPageData getCreateImageUploadUrlPageData() {
+        verifyPrivileges();
```

Comment:
This is not done yet. There are two places where this fix needs to be applied and only one is done.